build.rs

```diff
@@ -0,0 +1,2 @@
+/// Dummy build.rs so `OUT_DIR` is populated.
+fn main() {}
```

Comment:
do we have to have that? Is there no other way to achieve what we want without a `build.rs`?

I don't think its a big deal  in this case, but I am generally wary of using `build.rs` files unless it is absolutely necessary (as this is a common thing that makes downstream compilation optimizations worse.)

Reply:
I am not aware of any other way to populate the `OUT_DIR`, moving to `tmpdir()` we could remove the `build.rs` again

Reply:
@drahnr let's use tempdir then pls (I think we already have it in the tests..).

Reply:
Done